package.json

```diff
@@ -7,14 +7,13 @@
     "build": "webpack --config webpack.prod.js",
     "preinstall": "bin/preinstall.js"
   },
-  "dependencies": {},
   "devDependencies": {
+    "@fortawesome/fontawesome-free": "^5.15.2",
```

Comment:
is `fortawesome` a typo?

Reply:
This has confused me as well in the past, it is not a typo:
https://github.com/FortAwesome/Font-Awesome

Reply:
I thought it was a typo at first too lol